app/views/no_vat_repayments.scala.html

```diff
@@ -16,17 +16,32 @@
 
 @import config.ViewConfig
 @import uk.gov.hmrc.play.views.html._
+@import model.des.BankDetails
 
 @this(
 mainTemplate: views.html.main_template
```

Comment:
Instead of injecting template and bunch of smaller view partials it's better to inject one container with all view partials. 
We did it in pay-frontend and discussed it recently on tech and architecture meeting. 

```
@this(
    viewHelpers: ViewHelpers
)
...
  viewHelpers.main_template( ...)

//in another file

import uk.gov.hmrc.play.views.html.helpers._
import uk.gov.hmrc.play.views.html.layouts._

@Singleton
class ViewsHelpers @Inject() (
  
  mainTemplate: views.html.main_template,

  //copied from uk.gov.hmrc.play.views.html.helpers
  val address: Address,
  val dateFields: DateFields,
  val dateFieldsFreeInline: DateFieldsFreeInline,
  val dateFieldsFreeInlineLegend: DateFieldsFreeInlineLegend,
  val dateFieldsFreeYearInline: DateFieldsFreeYearInline,
  val dateFieldsFreeYear: DateFieldsFreeYear,
  val dateFieldsInline: DateFieldsInline,
  val dropdown: Dropdown,
  val errorInline: ErrorInline,
  val errorNotifications: ErrorNotifications,
  val errorSummary: ErrorSummary,
  val fieldGroup: FieldGroup,
  val form: FormWithCSRF,
  val input: Input,
  val inputRadioGroup: InputRadioGroup,
  val reportAProblemLink: ReportAProblemLink,
  val singleCheckbox: SingleCheckbox,
  val textArea: TextArea,

  //copied from uk.gov.hmrc.play.views.html.layouts
  val article: Article,
  val attorneyBanner: AttorneyBanner,
  val betaBanner: BetaBanner,
  val footer: Footer,
  val euExitLinks: EuExitLinks,
  val footerLinks: FooterLinks,
  val head: Head,
  val headerNav: HeaderNav,
  val loginStatus: LoginStatus,
  val mainContent: MainContent,
  val mainContentHeader: MainContentHeader,
  val optimizelySnippet: OptimizelySnippet,
  val gtmSnippet: GTMSnippet,
  val serviceInfo: ServiceInfo,
  val sidebar: Sidebar,

  //our partials
  endSurvey: uk.gov.hmrc.payment.views.html.barclays.partial.end_survey
)
```

Reply:
Done